### PR TITLE
Mail request status

### PR DESCRIPTION
**Pull request by sanjanagoli**
### Description
Mail Request Status

### Context
Request status with state variables depending on pending or confirmed order

### Issue
#120 

### Screenshot
Screen Shot 2019-04-12 at 11 48 14 PM

### CODE REVIEW

File: screens/RequestStatus.js [hunk 1]
```diff
@@ -1,127 +1,103 @@
...
 
+
+export default class RequestProfile extends React.Component { 
   static navigationOptions = {
     header: null,
     };
+  
+  
+  statusUpdate = () => {
+    //update when page is made
```
Comment (ijemmao) [inline screens/RequestStatus.js]: Remove comments

File: screens/RequestStatus.js [hunk 2]
```diff
@@ -1,127 +1,103 @@
...
-              <Text style={styles.browseText}>Browse Other Options</Text>
+              <Text style={styles.browseText}>See Other Deliveries</Text>
+            </TouchableOpacity>
+            <View style={styles.mainView}>
+              <View style={styles.whiteCard}>
+                <View style={styles.itemBox}>
+                  <Text style={styles.itemText}>Waiting for Confirmation</Text>
+                </View>
+                <View style={{textAlign: 'center'}}>
+                  <Text style={styles.descriptionText}>Come back when {this.state.item.firstName}'s mail request is confirmed!</Text>
```
Comment (ijemmao) [inline screens/RequestStatus.js]: Escape apostrophes

**Comment by sanjanagoli** [comment]
Changes made!

**Comment by ijemmao** [comment]
@sanjanagoli Address the remaining conflicts

**Comment by ijemmao** [comment]
@sanjanagoli You have remaining conflicts!

**Pull request merged**